src/program/lwaftr/query/query.lua

```diff
@@ -24,8 +23,24 @@ local function sort (t)
    return t
 end
 
-local function is_counter_name (name)
-   return lwcounter.counter_names[name] ~= nil
+local function counter_names ()
```

Comment:
This function is duplicated in check/util.lua. Likely is worth to put it somewhere like in `apps/lwaftr/lwutil.lua`.